tests/v2/validation/rbac/rbac.go

```diff
@@ -170,3 +172,36 @@ func deleteLabels(labels map[string]string) {
 		}
 	}
 }
+
+func convertSetting(steveAdminClient *v1.Client, globalSetting *v1.SteveAPIObject) (*v3.Setting, error) {
+	updateSetting := &v3.Setting{}
+	err := v1.ConvertToK8sType(globalSetting.JSONResp, updateSetting)
+	if err != nil {
+		return nil, err
+	}
+	return updateSetting, err
+}
+
+func listGlobalSettings(steveclient *v1.Client) (projectNames []string, err error) {
+	globalSettings, err := steveclient.SteveType("management.cattle.io.setting").List(nil)
+	if err != nil {
+		return nil, err
+	}
+
+	settingsNameList := make([]string, len(globalSettings.Data))
+	for idx, setting := range globalSettings.Data {
+		settingsNameList[idx] = setting.Name
+	}
+	sort.Strings(settingsNameList)
+	return settingsNameList, err
+}
+
+func editGlobalSettings(steveclient *v1.Client, globalSettingByID *v1.SteveAPIObject, value string) (*v1.SteveAPIObject, error) {
+	updateSetting, err := convertSetting(steveclient, globalSettingByID)
+	if err != nil{
+		return nil, err
+	}
+	updateSetting.Value = value
```

Comment:
nit; can this have an extra space after the error checking? Just to give it some "breathing room".